contracts/near/eth2-client/src/lib.rs

```diff
@@ -603,8 +614,6 @@ impl Eth2Client {
     }
 
     fn is_light_client_update_allowed(&self) {
```

Comment:
`#[pause(except(owner, self))]` ?

Reply:
This is an internal method, so the pausing check is moved to the top-level method `submit_beacon_chain_light_client_update`